packages/rest/README.md

```diff
@@ -46,32 +45,23 @@ The rest package is configured by passing a `rest` property inside of your
 Application options.
 
 ```ts
-  const app = new Application({
+  const app = new RestApplication({
     rest: {
       port: 3001
     }
   });
-  app.component(RestComponent);
 ```
 
 ### `rest` options
 
 | Property | Type | Purpose |
 |----------|------|---------|
 | port | number | Specify the port on which the RestServer will listen for traffic. |
-| secure (NOT YET IMPLEMENTED) | HttpsConfiguration | Provide a collection of secure key materials to the RestServer, which switches it to HTTPS mode. |
-| sequence | SequenceHandler | Use a custom SequenceHandler to change the behaviour of the RestServer for the request-response lifecycle. |
-
-#### HttpsConfiguration
-<!-- TODO(@kjdelisle): Define the contents of this type for users. -->
-
-#### SequenceHandler
-<!-- TODO(@kjdelisle): Point to the request-response lifecycle doc
-(or migrate it here?) -->
+| sequence | SequenceHandler | Use a custom SequenceHandler to change the behavior of the RestServer for the request-response life cycle. |
 
 ## Contributions
 
-- [Guidelines](https://github.com/strongloop/loopback-next/wiki/Contributing#guidelines)
+- [Guidelines](https://github.com/strongloop/loopback-next/blob/master/docs/DEVELOPING.md)
```

Comment:
I think it'd be better to link to https://github.com/strongloop/loopback-next/blob/master/docs/CONTRIBUTING.md which serves as the overall guideline and points to our `DEVELOPING` doc as needed.